modulefiles/module_base.hera.lua

```diff
@@ -41,11 +41,10 @@ load(pathJoin("py-xarray", (os.getenv("py_xarray_ver") or "None")))
 setenv("WGRIB2","wgrib2")
 setenv("UTILROOT",(os.getenv("prod_util_ROOT") or "None"))
 
---prepend_path("MODULEPATH", pathJoin("/scratch1/NCEPDEV/global/glopara/git/prepobs/v" .. (os.getenv("prepobs_run_ver") or "None"), "modulefiles"))
-prepend_path("MODULEPATH", pathJoin("/scratch1/NCEPDEV/global/glopara/git/prepobs/feature-GFSv17_com_reorg_log_update/modulefiles"))
+prepend_path("MODULEPATH", pathJoin("/scratch1/NCEPDEV/global/glopara/git/prepobs/feature-hera-rocky8/modulefiles"))
```

Comment:
We have completed the prepobs PR into our prepobs `dev/gfsv17` branch. Please update this to change `feature-hera-rocky8` to `dev-gfsv17`.

For reference, the full install path is: `/scratch1/NCEPDEV/global/glopara/git/prepobs/dev-gfsv17`